habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/PyenvAndPoetrySetup.java

```diff
@@ -147,6 +193,24 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             log.info("Checking for updates to poetry-monorepo-dependency-plugin...");
             poetryHelper.installPoetryPlugin("poetry-monorepo-dependency-plugin");
         }
+
+        String username = null;
+        String password = null;
+
+        if (StringUtils.isNotEmpty(this.pypiRepoId)) {
+            username = HabushuUtil.findUsernameForServer(this.settings, this.pypiRepoId);
+            password = HabushuUtil.findPasswordForServer(settings, pypiRepoId, decryptPassword);
```

Comment:
minor:  Just for consistency, can you use --> this.settings and this.pypiRepoId and  this.decryptPassword.  Or, alternatively, change the line above to drop --> this.blahblah

Reply:
Done.